app/assets/src/components/views/amr_heatmap/AMRHeatmapView.jsx

```diff
@@ -286,6 +295,7 @@ export default class AMRHeatmapView extends React.Component {
       samplesWithAMRCounts,
       selectedOptions,
       samplesMetadataTypes,
+      maxValues,
```

Comment:
You should initialize maxValues at the beginning of the components.
Helps with readability and understand variables used through out the code.